test/e2e/resources/alias_simple.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kms.services.k8s.aws/v1alpha1
+kind: Alias
+metadata:
+  name: $ALIAS_NAME
+spec:
+  name: alias/$ALIAS_NAME
```

Comment:
any need for the `alias/` prefix here? just curious...

Reply:
Server side validation requirement that all alias names begin with `alias/` - https://docs.aws.amazon.com/sdk-for-go/api/service/kms/#CreateAliasInput